lib/services/northBound/deviceProvisioningServer.js

```diff
@@ -65,6 +66,15 @@ var async = require('async'),
         explicitAttrs: 'explicitAttrs'
     };
 
+
+if(config.relaxTemplateValidation){
```

Comment:
Code style:

```suggestion
if (config.relaxTemplateValidation) {
```

Reply:
If only there was a way of dealing with these whitespace issues automatically? https://github.com/telefonicaid/iotagent-node-lib/pull/831

Reply:
Fixed 62239f2